ui/src/app/review/assessment/sections/section.component.ts

```diff
@@ -64,7 +64,10 @@ export class ExamSectionComponent implements OnInit, AfterViewInit {
     // Since the essay questions are the only ones that need to be evaluated and the rest of the questions are evaluated automatically.
     getReviewProgress = () => {
         return this.section.sectionQuestions.filter((q: ExamSectionQuestion) => {
-            return q.question.type !== 'EssayQuestion' || q.essayAnswer?.evaluatedScore;
+            return (
```

Comment:
Voisiko tässä käyttää shared/miscellaneous/helpers-tiedostosta funktiota isNumber jos siis on tarkoitus erottaa onko arvo 0, undefined vai null?

`export const isNumber = (a: unknown): a is number => typeof a === 'number';`

eli sen jälkeen tuo olisi `q.question.type !== 'EssayQuestion' || isNumber(q.essayAnswer?.evaluatedScore)`

Reply:
No nythän siitä tuli siisti